sormas-app/app/src/main/java/de/symeda/sormas/app/backend/common/AdoDtoHelper.java

```diff
@@ -21,7 +21,7 @@
 import com.j256.ormlite.logger.Logger;
 import com.j256.ormlite.logger.LoggerFactory;
 
-import org.springframework.util.CollectionUtils;
+import org.apache.commons.collections.CollectionUtils;
```

Comment:
Switch to `org.apache.commons.collections4.CollectionUtils`, also in `SormasToSormasShareRequestService`  and `ShareRequestInfoService`.